new-contract-interaction/agent-config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This file can be removed from the repository.  We are using the `agent-config.json.example` file as the template for the user to copy and modify.